roles/rust-foundation-project-director.md

```diff
@@ -0,0 +1,35 @@
+# Role Description: Rust Foundation Project Director
+
+Please refer to [Rust Foundation Director Roles & Responsibilities](TODO) for components of the role description that apply to all Foundation Board Directors.
+
+## Goals
+
+- Represent the needs and interests of the Rust Project at large within the Rust Foundation board
+- Ensure communication between the Rust Project and Foundation is robust and high fidelity.
+- TODO Fill out this list
+
+## Time Budget
+
+- 2-5 hours per week
+
+## Tasks / Activities / Responsibilities
+
+- [Abide by the Roles & Responsibilites of Foundation Board Members](https://foundation.rust-lang.org/static/board-director-role-description.pdf)
+- Attend Rust Foundation board meetings
+- Help prepare and present the "Project Director Update" for each board meeting
+- Meet with the Rust Foundation staff on a regular basis
+- Work with the Rust Leadership Council and Rust Foundation staff on refining the role of Project Director for the future
+- TODO Fill out this list
+
+## Qualities of a Council member (required skills)
+
+- Sufficient time and energy to dedicate to the needs of the Rust Foundation and the Rust Project.
+- An interest in helping with the topics of Project operations and Project governance.
+- Broad awareness of the needs of the Project outside of their teams or areas of active contribution.
```

Comment:
This one seems redundant with the next one. And I'm not sure how I would define "broad awareness" if I were trying to assess if I were a good fit. A lot of that awareness comes with the job, and the council will be there to assist. So I'd argue that while a director does need some sense of the needs of the Project, the breadth comes from being willing to consider the needs and viewpoints of others (see below).

Reply:
```suggestion
```